module/applications/actor/character-sheet-2.mjs

```diff
@@ -346,11 +346,12 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
 
     // Apply special context menus for items outside inventory elements
     const featuresElement = html[0].querySelector(`[data-tab="features"] ${this.options.elements.inventory}`);
-    if ( featuresElement ) {
-      new ContextMenu5e(html, ".pills-lg [data-item-id], .favorites [data-item-id], .facility[data-item-id]", [], {
-        onOpen: (...args) => featuresElement._onOpenContextMenu(...args)
-      });
-    }
+    if ( featuresElement ) new ContextMenu5e(
+      game.release.generation < 13 ? html : html[0],
```

Comment:
`ContextMenu` has actually supported passing an HTMLElement as the first argument for a while, even in v12.

```suggestion
      html[0],
```